### PR TITLE
Add Second Keynote Speaker

### PR DESCRIPTION
**Pull request by ananyo2012**
Zainab Bawa CEO of HasGeek

### CODE REVIEW

File: _data/speakers.yml [hunk 1]
```diff
@@ -27,14 +27,24 @@ sarabjot_singh:
...
-#         instagram: instagram.url
-#     about: <p>The second speaker. Text about the second speaker. Text about the second speaker. Text about the second speaker. Text about the second speaker. Text about the second speaker. Text about the second speaker. Text about the second speaker.</p><p class=mb-0>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum, nemo, illo, doloribus voluptatem dolorum harum nesciunt delectus fuga qui veniam itaque reprehenderit eos error sunt rem culpa facilis! Quo, autem.</p>
+zainab_bawa:
+  keynote: true
+  name: Zainab Bawa
+  job: CEO & Co-founder
+  company: HasGeek
+  photo: Zainab.jpg
+  social:
+    twitter: https://twitter.com/zainabbawa
```
Comment (inovizz) [inline _data/speakers.yml:37]: Can you add her linkedin profile as well - https://www.linkedin.com/in/zainabbawa

File: _data/speakers.yml [hunk 2]
```diff
@@ -27,14 +27,24 @@ sarabjot_singh:
...
-#     photo: speaker-2
-#     social:
-#         linkedin: linkedin.url
-#         github: github.url
-#         instagram: instagram.url
-#     about: <p>The second speaker. Text about the second speaker. Text about the second speaker. Text about the second speaker. Text about the second speaker. Text about the second speaker. Text about the second speaker. Text about the second speaker.</p><p class=mb-0>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eum, nemo, illo, doloribus voluptatem dolorum harum nesciunt delectus fuga qui veniam itaque reprehenderit eos error sunt rem culpa facilis! Quo, autem.</p>
+zainab_bawa:
+  keynote: true
+  name: Zainab Bawa
+  job: CEO & Co-founder
```
Comment (inovizz) [inline _data/speakers.yml:33]: As per her email, its - Founder & CEO

File: _data/speakers.yml (hunk 1 above)
Comment (ananyo2012) [reply _data/speakers.yml:37]: Just merged it. Adding a hotfix

File: _data/speakers.yml (hunk 2 above)
Comment (ananyo2012) [reply _data/speakers.yml:33]: Her medium account said Co-founder. Will update it.

**Pull request merged**